### DISCUSSION TITLE
To use or not to use UseCases................

### DISCUSSION THREAD

**Discussion by kibettheophilus** [droidconKE/droidconKE2022Android#81] [Q&A]
This is a discussion on whether to use usecases in the droidconKE app.

 Fell free to share your ideas on why we should or should not use useacases. 

<img src ="https://c.tenor.com/h1OM-S8qtjUAAAAC/what-are-your-thoughts-on-this-whats-your-opinion.gif"/>

**Comment by wangerekaharun** [comment]
Not to use cases wins
